content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -29,6 +29,19 @@ Kubernetes, on Linux:
 - [Docker Engine](#docker)
 - [Mirantis Container Runtime](#mcr)
 
+{{< note >}}
+Dockershim, the portion of code in Kubernetes that provided direct
+integration with Docker in prior releases, was removed from Kubernetes
+version 1.24. This removal was announced as a [deprecation in Kubernetes v 1.20](
+/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
+You can check out this [documentation](
+/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/)
+to understand how this deprecation might affect you. For migrating from
```

Comment:
Leading "to migrate from dockershim" looks redundant/tautological to me:

For migrating from dockershim ... **to migrate from dockershim.**

Reply:
done @bart0sh

Reply:
@PranshuSrivastava thanks, looks better now.